torchtext/datasets/yahooanswers.py

```diff
@@ -42,13 +42,27 @@ def YahooAnswers(root: str, split: Union[Tuple[str], str]):
 
     cache_dp = url_dp.on_disk_cache(
         filepath_fn=lambda x: os.path.join(root, _PATH),
-        hash_dict={os.path.join(root, _PATH): MD5}, hash_type="md5"
+        hash_dict={os.path.join(root, _PATH): MD5},
+        hash_type="md5"
     )
     cache_dp = GDriveReader(cache_dp).end_caching(mode="wb", same_filepath_fn=True)
     cache_dp = FileOpener(cache_dp, mode="b")
```

Comment:
nit: should we rename to `cache_compressed_dp` to be consistent with the [AmazonReviewPolarity PR](https://github.com/pytorch/text/pull/1527/files#diff-daeb7a9b451f1cb20240d2fda55074b6f03d48849646c931c513050af8eff1a5R43-R46)?